cypress/integration/visitor/loginPage/signIn.spec.js

```diff
@@ -1,19 +1,38 @@
 describe('Sign In', () => {
   beforeEach(() => {
-    cy.fixture('users').then(users => {
+    cy.fixture('users').then((users) => {
       this.users = users;
     });
   });
 
-  it('Visitor singns in with valid credentials', () => {
+  afterEach(() => {
```

Comment:
Do we really need to sign out after each test case?

Reply:
@nadezhdakharchuk We need do this, when beforeEach or in test case login command is executed. Because if we want to have the access to login page we must be unauthorized

Reply:
Ok. Let's add the logout command to `support/commands`